From da2cee2a117e5378a36a2cf57da360355c97dde3 Mon Sep 17 00:00:00 2001
From: Aelin <aelin@postmarketos.org>
Date: Tue, 14 Apr 2026 09:37:25 +0200
Subject: [PATCH 03/10] Allow systemd-hwdb to log to syslog

Resolves the following denial:
[    8.000469] audit: type=1400 audit(1776152202.643:4): avc:  denied  { sendto } for  pid=342 comm="systemd-hwdb" path="/run/systemd/journal/socket" scontext=system_u:system_r:systemd_hwdb_t:s0 tcontext=system_u:system_r:syslogd_t:s0 tclass=unix_dgram_socket permissive=0
---
 policy/modules/system/systemd.te | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/policy/modules/system/systemd.te b/policy/modules/system/systemd.te
index 1f323ddb6..73738e785 100644
--- a/policy/modules/system/systemd.te
+++ b/policy/modules/system/systemd.te
@@ -1426,6 +1426,8 @@ dev_write_kmsg(systemd_hwdb_t)
 
 kernel_dgram_send(systemd_hwdb_t)
 
+logging_send_syslog_msg(systemd_hwdb_t)
+
 systemd_read_efivarfs(systemd_hwdb_t)
 
 ########################################
-- 
2.55.0

